recommendation/dlrm/pytorch/docker_cpu/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN cd /opt && \
     echo "conda activate base" >> ~/.bashrc && \
     conda config --set always_yes yes --set changeps1 no
 
-RUN conda install pytorch-cpu torchvision-cpu -c pytorch
+RUN conda install pytorch torchvision cpuonly -c pytorch
 RUN pip install future pillow onnx opencv-python-headless tensorflow onnxruntime
 RUN pip install Cython && pip install pycocotools
```

Comment:
Does this reference code depend on tensorflow (since this a reference implementation for pytorch)?

Reply:
Could you use PR code from https://github.com/dkorchevgithub/inference/tree/master/recommendation

The container has tensorflow in it, but we will need to do more testing with it. The scripts provided are for pytorch at this point. Please feel free to test it and let us know if you find new issues. Thank you for your feedback.